src/components/Chip.tsx

```diff
@@ -20,13 +20,14 @@ export interface ChipProps<X> {
   title?: string;
   xss?: X;
   type?: ChipType;
+  compact?: boolean;
 }
 
 /** Kinda like a chip, but read-only, so no `onClick` or `hover`. */
 export function Chip<X extends Only<Xss<Margin>, X>>({ type = ChipTypes.neutral, ...props }: ChipProps<X>) {
-  const { text, title = text, xss = {} } = props;
+  const { text, title = text, xss = {}, compact: propsCompact } = props;
   const { fieldProps } = usePresentationContext();
-  const compact = fieldProps?.compact;
+  const compact = propsCompact ?? fieldProps?.compact;
```

Comment:
Does this `compact` variable give us the sizes we want? If so, then `fieldProps.compact` should be set to `true` when in a table... 🤔

Reply:
it does give the size showed on the design 👍 the table is picking it up as compact i don't know from where, but is rendering with the `xs` size

Reply:
Okay awesome. So was this change even necessary to accomplish your ticket?

Reply:
yes to match the design mock, i also had to add 2px padding for the text content to compensate for the chip 2px padding that shows the background color, but that was necessary to prevent the card height to change between edit and read mode

Reply:
My hope is that you won't need to make any changes to the Chip component from `internal-frontend` and we can use it as is.